spec/models/content_configuration_spec.rb

```diff
@@ -12,7 +12,7 @@
     end
 
     def image_exist?(default_url)
-      File.exist?(Rails.public_path.join(default_url).to_s)
+      File.exist?(Rails.public_path.to_s + default_url)
```

Comment:
I don't think this is necessary, they are more or less equivalent, but the `join` version is better as it will handle properly the use of / or \ in path.

Reply:
They generate two different URLs causing the test to fail. The original line generates the url to access it via web I believe. The corrected version gives you the local path.

Reply:
Ahh, `default_url` is being interpreted as an absolute URL because it has a forward slash at the beginning of it.

Reply:
What about this :
```suggestion
      File.exist?(File.join(Rails.public_path, default_url))
```

Reply:
Rspec passes, but Rubocop still complains:
```
spec/models/content_configuration_spec.rb:15:19: C: [Correctable] Rails/RootPathnameMethods: Rails.public_path is a Pathname so you can just append #join.
      File.exist?(File.join(Rails.public_path, default_url))
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

1484 files inspected, 1 offense detected, 1 offense autocorrectable
```

Reply:
That's a tricky one, it comes down to `File.join` and and `Pathname#join` not behaving the same when there is a leading "/".  Let's go with your last solution, as it behaves the same has the original code, ie having a leading "/" or not would return the same file path.  Maybe just add a comment to mention why we are doing this.